### PR TITLE
TimeSeries: Make timeseries the default panel and remove beta state

### PR DESCRIPTION
**Pull request by torkelo**
Replaces https://github.com/grafana/grafana/pull/30919

Not sure we need the banner?

Think we could need like a generic panel docs info / docs link at the top of the options pane so users can more easily find info on the new panel

### CODE REVIEW

**Comment by torkelo** [comment]
This requires some tweaks to panel sort order as well, going to do that after https://github.com/grafana/grafana/pull/33633 is merged

**Review by dprokop — APPROVED** [review]
Works good. I really like the descriptions on the cards!

**Comment by dprokop** [comment]
An idea - maybe we could list the deprecated panels at the bottom of the list?

**Comment by torkelo** [comment]
> An idea - maybe we could list the deprecated panels at the bottom of the list?

@dprokop 
deprecated panels (old table panel) are not shown in the list. The graph panel is a bit special and think we should allow users to pick for  a release or two before we get more solid feedback with time series as the default

File: public/app/plugins/panel/graph/plugin.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "type": "panel",
-  "name": "Graph",
+  "name": "Graph (old)",
```
[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/plugins/panel/graph/plugin.json:3]: think I prefer old as an easier to understand term for nontechnical users.

[the remark this answers is not in the document]
Comment (dprokop) [reply public/app/plugins/panel/graph/plugin.json:3]: sounds good

**Review by mckn — APPROVED** [review]
LGTM!

**Pull request merged**